image/bif.ksy

```diff
@@ -0,0 +1,67 @@
+meta:
+  id: bif
+  file-extension: bif
+  endian: le
+  title: Roku Base Index Frames (BIF) image archive format
+  license: MIT
+seq:
```

Comment:
It may be useful for it to have more `meta`, I mean `application:` describing the apps this format is using in and `doc` and `doc-ref`.